decidim-core/app/cells/decidim/flag_modal_cell.rb

```diff
@@ -21,7 +21,9 @@ def cache_hash
     private
 
     def frontend_administrable?
-      model.can_be_administred_by?(current_user) && (model.respond_to?(:official?) && !model.official?)
+      author.respond_to?(:nickname) &&
+        model.can_be_administred_by?(current_user) &&
```

Comment:
We could fix the typo in this PR too: The word "administred" doesn't exist, it should be "administered"